sdks/python/src/opik/api_objects/dataset/dataset.py

```diff
@@ -76,6 +94,21 @@ def insert(
                 dataset_name=self._name, items=batch
             )
 
+    def _sync_hashes(self) -> None:
+        """Updates all the hashes in the dataset"""
+        LOGGER.debug("Start hash sync in dataset")
+        all_items = self.get_all_items()
```

Comment:
Minor: This call seemed to have worked well per your testing, which is great because it's internally paginated. In the future it might consume too much space, but this is fine and no action point so far.